example_test.go

```diff
@@ -159,31 +171,31 @@ func Example_decodePSAEndorsementSoftwareComponent() {
 		"software-name": "Roadrunner boot loader",
 		"software-version": "1.0.0",
 		"entity": [
-		  {
+		{
 			"entity-name": "ACME Ltd",
 			"reg-id": "acme.example",
 			"role": [
-			  "tagCreator",
-			  "aggregator"
+			"tagCreator",
+			"aggregator"
 			]
-		  }
+		}
 		],
 		"link": [
-		  {
+		{
 			"href": "example.acme.roadrunner-hw-v1-0-0",
 			"rel": "psa-rot-compound"
-		  }
+		}
 		],
 		"payload": {
-		  "resource": [
+			"resource": [
 			{
-			  "type": "arm.com-PSAMeasuredSoftwareComponent",
-			  "arm.com-PSAMeasurementValue": "sha-256:YWFiYi4uLmVlZmY=",
-			  "arm.com-PSASignerId": "sha-256:NTE5Mi4uLjEyMzQ="
+				"type": "arm.com-PSAMeasuredSoftwareComponent",
+				"arm.com-PSAMeasurementValue": "sha-256:YWFiYi4uLmVlZmY=",
+				"arm.com-PSASignerId": "sha-256:NTE5Mi4uLjEyMzQ="
 			}
-		  ]
+			]
 		}
-	  }`)
+	}`)
```

Comment:
nothing seems to have substantially changed here except for the indentation.

Reply:
the indentation change was not on purpose. It must have slipped my mind.
Fixed

Reply:
thanks!